classes/Imagifybeat/Actions.php

```diff
@@ -66,18 +68,13 @@ public function add_requirements_to_response( $response, $data ) {
 			return $response;
 		}
 
-		$is_blocked    = \Imagify_Requirements::is_imagify_blocked();
-		$is_api_up     = \Imagify_Requirements::is_api_up();
-		$is_key_valid  = \Imagify_Requirements::is_api_key_valid();
-		$is_over_quota = \Imagify_Requirements::is_over_quota();
-
 		$response[ $imagifybeat_id ] = [
-			'curl_missing'          => ! \Imagify_Requirements::supports_curl(),
-			'editor_missing'        => ! \Imagify_Requirements::supports_image_editor(),
-			'external_http_blocked' => $is_blocked,
-			'api_down'              => $is_blocked || ! $is_api_up,
-			'key_is_valid'          => ! $is_blocked && $is_api_up && $is_key_valid,
-			'is_over_quota'         => ! $is_blocked && $is_api_up && $is_key_valid && $is_over_quota,
+			'curl_missing'          => ! Requirements::supports_curl(),
+			'editor_missing'        => ! Requirements::supports_image_editor(),
+			'external_http_blocked' => Requirements::is_imagify_blocked(),
+			'api_down'              => ! Requirements::is_api_up(),
+			'key_is_valid'          => Requirements::is_api_key_valid(),
+			'is_over_quota'         => Requirements::is_over_quota(),
```

Comment:
Let's be consistent. [Here](https://github.com/wp-media/imagify-plugin/pull/479/files#diff-c2c4f325ab724701d7fae667b2701ffaR83) the code uses the full class name instead of an alias. Typically we only want to use an alias _when_ there's a conflict with class names.

I'd suggest removing the `as Requirements` alias and using the full class name on lines 72-77.

Reply:
We also use alias to shorten our class names and make the code easier to read, like using `Foobar` instead of `\SubVeryLong1\SubVeryLong2\SubVeryLong3\SubVeryLong4\Foobar`.